tests/queries/0_stateless/02972_parallel_replicas_cte.sql

```diff
@@ -45,31 +45,31 @@ FROM
     SELECT c + 1
     FROM
     (
-        WITH filtered_groups AS (SELECT a FROM pr_1 WHERE a >= 10000)
+        WITH filtered_groups AS (SELECT a FROM pr_1 WHERE a >= 100)
         SELECT count() as c FROM pr_2 INNER JOIN filtered_groups ON pr_2.a = filtered_groups.a
     )
 )
 SETTINGS allow_experimental_parallel_reading_from_replicas = 1, parallel_replicas_for_non_replicated_merge_tree = 1, cluster_for_parallel_replicas = 'test_cluster_one_shard_three_replicas_localhost', max_parallel_replicas = 3;
 
-CREATE TABLE numbers_1e6
+CREATE TABLE numbers_1e3
```

Comment:
do we need to reduce it so much?

Reply:
Probably not, but at this point, with the CI taking ages on slow tests, I'd rather err towards reducing too much vs having too many rows.